tests/e2e-autoscale/autoscale/04-delete.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kuttl.dev/v1beta1
```

Comment:
we're on chainsaw now :)

Reply:
Thanks for the heads-up. I've just submitted the e2e using Chainsaw instead.